### PR TITLE
fix: Update Metatag patch for 8.x-1.22

### PR DESCRIPTION
**Pull request by alex-ahumada**
Patch "Support JSON API, REST, GraphQL and custom normalizations" fails in 8.x-1.22, this updated patch works fine with latest version.

### CODE REVIEW

File: drupal/composer.json [hunk 1]
```diff
@@ -120,12 +120,12 @@
         "pantheon-systems/drupal-integrations"
       ],
       "locations": {
-          "web-root": "./web"
+        "web-root": "./web"
       },
       "file-mapping": {
-          "[project-root]/.editorconfig": false,
```
Comment (shadcn) [inline drupal/composer.json:126]: @alex-ahumada It looks like a few unrelated changes got in. Can we fix those please?

Sorry for the late review.

**Review by shadcn — CHANGES_REQUESTED** [review]

File: drupal/composer.json (hunk 1 above)
Comment (alex-ahumada) [reply drupal/composer.json]: Done @shadcn! Sorry, my WebStorm config removed some whitespace.

**Comment by alex-ahumada** [comment]
This got fixed on 0b03a72. Closing PR.
